docs/source/cpp/compute.rst

```diff
@@ -140,7 +140,7 @@ Aggregations
 +--------------------------+------------+--------------------+-----------------------+--------------------------------------------+
 | min_max                  | Unary      | Numeric            | Scalar Struct  (1)    | :struct:`MinMaxOptions`                    |
 +--------------------------+------------+--------------------+-----------------------+--------------------------------------------+
-| mode                     | Unary      | Numeric            | Scalar Struct  (2)    |                                            |
+| mode                     | Unary      | Numeric            | Scalar Struct  (2)    | :struct:`ModeOptions`                      |
```

Comment:
Replace "Scalar Struct" with "Struct".

Reply:
Mode now returns an array of struct. Should it be "Struct Array"? or simply "Struct'.
Should I also change above line (min_max) to "Struct"? or leave it "Scalar Struct".

Reply:
`min_max` returns a scalar, AFAIK.

Reply:
And `mode` should be changed to "Struct".

Reply:
Thanks. Done.